src/components/panes/declared-variables-pane.jsx

```diff
@@ -8,6 +8,8 @@ import SidePane from './side-pane'
 import { DeclaredVariable } from './declared-variable'
 import tasks from '../../actions/task-definitions'
 
+const doNotShow = new Set(['CodeMirror'])
```

Comment:
did you know that you've already created equivalent functionality elsewhere in the code? see `initialVariables` in `notebook-reducer.js`. ;-)

Reply:
I am half-aware, but figured it somehow got lost during one of the refactors since the declared variables pane doesn't functionally use any of it. I'll take a second look here and remove this if not.